File: src/main/resources/output/IFC4X3_csharp/Entities/IfcRoot/IfcRelationship/IfcRelAssigns/IfcRelAssigns.cs
public class IfcRelAssigns : IfcRelationship
{
    public SET [1:?] OF IfcObjectDefinition RelatedObjects { get; set; }
    public IfcStrippedOptional RelatedObjectsType { get; set; }

    // === EXTENDED BY ===
    // IfcRelAssignsToActor
    // IfcRelAssignsToControl
    // IfcRelAssignsToGroup
    // IfcRelAssignsToProcess
    // IfcRelAssignsToProduct
    // IfcRelAssignsToResource
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcRoot/IfcRelationship/IfcRelAssigns/IfcRelAssigns.vb
Public Class IfcRelAssigns Inherits IfcRelationship
    Public RelatedObjects As SET [1:?] OF IfcObjectDefinition
    Public RelatedObjectsType As IfcStrippedOptional

    ' === EXTENDED BY ===
    ' IfcRelAssignsToActor
    ' IfcRelAssignsToControl
    ' IfcRelAssignsToGroup
    ' IfcRelAssignsToProcess
    ' IfcRelAssignsToProduct
    ' IfcRelAssignsToResource
End Class
